autofit/graphical/__init__.py

```diff
@@ -1,6 +1,13 @@
-from .declarative import ModelFactor, ModelFactorCollection
-from .factor_graphs import Factor, FactorGraph, AbstractFactor, FactorValue
-from .mean_field import FactorApproximation, MeanFieldApproximation
-from .messages import FixedMessage, NormalMessage, GammaMessage, AbstractMessage
-from .optimise import OptFactor, lstsq_laplace_factor_approx
-from .sampling import ImportanceSampler, project_factor_approx_sample
+from .declarative import ModelFactor, ModelFactorCollection
+from .factor_graphs import \
```

Comment:
If you're importing lots of stuff from a single module it's sometimes nice to alias the module and use that.
e.g.
```python
from . import factor_graphs as fg

fg.Factor

Reply:
yeah I can see that, but in this case it's useful to directly expose those imports so only need to import `autofit.graphical`

```
import autofit.graphical as graph

graph.Factor
```

Reply:
Oh yeah you're totally right I didn't pay attention to the filename!